MAKE/Makefile.ukko_cuda

```diff
@@ -0,0 +1,118 @@
+# Makefile for the ukko Cluster at UH, using nvidia GPUs
+#
+# Recommended setup:
+# * Compile on an interactive node with the below modules;
+#   See the It4Science wiki or the Vorna guide
+#   https://github.com/fmihpc/analysator/wiki/Vorna-guide
+#   for the current best practice.
+# * Use mpirun to run, with the following (rather long) command line:
+#   mpirun -mca pml ucx --mca btl ^vader,tcp,openib -x UCX_NET_DEVICES=mlx5_0:1 -x UCX_TLS=rc,sm -x UCX_IB_ADDR_TYPE=ib_global $executable --run_config $configfile
+#
+# module load GCC/13.2.0
+# module load OpenMPI/4.1.6-GCC-13.2.0
+# module load PMIx/4.2.6-GCCcore-13.2.0
+# module load PAPI/7.1.0-GCCcore-13.2.0
+# module load CUDA
+
+# Oneliner for command line:
+# module purge; module load GCC/13.2.0; module load OpenMPI/4.1.6-GCC-13.2.0 ; module load PMIx/4.2.6-GCCcore-13.2.0; module load PAPI/7.1.0-GCCcore-13.2.0; module load CUDA
+
+CMP = mpicxx
```

Comment:
delete these lines as these CMP and LNK are not used

Reply:
Done.